apps/web/playwright/payment-apps.e2e.ts

```diff
@@ -200,4 +200,38 @@ test.describe("Payment app", () => {
     // Expect "Getting started with Paypal APP" to be displayed
     expect(await page.locator("text=Getting started with Paypal APP").first()).toBeTruthy();
   });
+
+  /**
+   * For now almost all the payment apps show display "This app has not been setup yet"
+   * this can change in the future
+   */
+  test("Should not display App is not setup already for non payment app", async ({ page, users }) => {
```

Comment:
New test that should validate non payment apps render properly. Only for now all payment apps display the new setup flow when credentials are not given yet so you can go to app/setup page and put them there.